test/files/rules/nounusedvariable-falsepositives.test.ts

```diff
@@ -0,0 +1,18 @@
+const fs = require("fs");
```

Comment:
can you investigate whether this change fixes #613 and #651 as well? if so, please add those tests to this file